code/mob/input.dm

```diff
@@ -202,6 +202,12 @@
 						last_move_trigger = ticker ? ticker.round_elapsed_ticks : 0 //Wire note: Fix for Cannot read null.round_elapsed_ticks
 						deliver_move_trigger(running ? "sprint" : m_intent)
 
+					// Tripping (the physical kind)
+					var/trip_chance = 5
```

Comment:
an alternative implementation could be to add a signal (`COMSIG_MOB_MOVE` or something) that's called here, and add tripping as a component that hooks into such a signal. This would be better because it would avoid the need to check traitholder on every single step of every single mob's `process_move`.

Reply:
In that case we should also fix the "Two Left Feet" trait, because that does it even more frequently.

I've personally no idea how to work with signals, however, and feel like I'd do a shoddy job, but the idea behind it seems better.